packages/server/graphql/types/ReflectPhase.ts

```diff
@@ -30,7 +30,7 @@ const ReflectPhase = new GraphQLObjectType<any, GQLContext>({
       type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(RetroPhaseItem))),
       description: 'The prompts used during the reflect phase',
       resolve: async ({promptTemplateId, teamId}, _args, {dataLoader}) => {
-        const phaseItems = await dataLoader.get('customPhaseItemsByTeamId').load(teamId)
+        const phaseItems = await dataLoader.get('reflectPromptsByTemplateId').load(teamId)
```

Comment:
this should take in a `templateId`, which will obviate the line below it

Reply:
do we still need the promptTemplateId?

Reply:
do we remove promptTemplateId from both ReflectPhase and ReflectTemplate?

Reply:
nope, i think both still need it

Reply:
ok got it i'll leave it in